packages/validator-bonds-cli-core/src/commands/manage/fundBond.ts

```diff
@@ -49,18 +49,26 @@ export function configureFundBond(program: Command): Command {
         '(default: wallet keypair)',
       parseWalletOrPubkey,
     )
+    .option(
+      '--compute-unit-limit <number>',
+      'Compute unit limit for the transaction (default value based on the operation type)',
```

Comment:
```suggestion
      'Compute unit limit for the transaction',
```
(non-blocking)
This seems to have a specific default value of `FUND_BOND_LIMIT_UNITS` instead of being based on the operation type, so this might be a copy-paste error.